motoko/exchange_rate/canister_ids.json

```diff
@@ -0,0 +1,11 @@
+{
+  "__Candid_UI": {
+    "ic": "a4gq6-oaaaa-aaaab-qaa4q-cai"
+  },
+  "exchange_rate": {
+    "ic": "za3g3-aiaaa-aaaao-qaaca-cai"
+  },
+  "exchange_rate_assets": {
+    "ic": "zh2ap-nqaaa-aaaao-qaacq-cai"
+  }
+}
```

Comment:
`dfx deploy --network ic` will try to deploy to these canisters. I think it's best to delete this file

Reply:
Hi @sesi200, I thought regular developers can't deploy to these canister_ids even if they know the id and have the source code, as they are not canister controllers. 

On the other hand, we are trying to announce the canister_ids in public documentation, so these are not hidden info even if I remove them from here. 

This file is mostly to make future deployments easier. If you strongly prefer to remove them from here, I'd be happy to remove.

Reply:
Looks like I didn't explain myself well enough, sorry.

> I thought regular developers can't deploy to these canister_ids even if they know the id and have the source code, as they are not canister controllers.

This is correct.

> On the other hand, we are trying to announce the canister_ids in public documentation, so these are not hidden info even if I remove them from here.

IMO (which may be wrong 😄) noone will discover the canister IDs of the example if you only put them into this file. I would actually put the links to the example deployment into the README

> This file is mostly to make future deployments easier.

I totally get that. My main concern is that if someone else (who isn't a controller of these canisters) tries to deploy this to mainnet, then it will _also_ try to deploy to these canister IDs and give `403 permission denied`. And if a beginner runs into this they won't think of checking `canister_ids.json` because they don't know how things work behind the scenes.

I'm ok leaving this in here, but I think it should at least be renamed (e.g. to `canister_ids.json_example_deployment`) so that a simple `dfx deploy --network ic` can work

Reply:
Thanks for the detailed explanation @sesi200 !  I renamed `canister_ids.json` to `canister_ids.json.example` as a middle ground.